app/views/submitted_content/_hyperlink.html.erb

```diff
@@ -12,9 +12,11 @@
     <%= f.hidden_field :participant_id, value: participant.id %>
     </br>
   <% end %>
-
+  
   <a href="#" id="urlLink" name="urlLink" onClick="toggleElement('url','links');return false;">hide links</a><br/>
-  <div id="url" style="">
+  <!-- Setting user-select to none prevents users from copy-pasting url links -->
+  <div id="url" style="user-select: none">
```

Comment:
This was changed so that users could not copy-paste links and put them into the browser bar, circumventing the tracking system.